api/routes/salary-data-router.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @summary This is the Salary Data Endpoint for SET, its purpose is to provide
+ *          perform CRUD operations on Salary Data
+ * @author dallascrichmond
+ */
+import express from 'express';
+import {
+    getSalaryDataById,
+    getSalaryDataByParams,
+    createSalaryData,
+    deleteSalaryData
+} from '../controllers/salary-data-controller';
+
+const router = express.Router();
+
+// CREATE salary data
+router.route('/salary')
+    .post(createSalaryData);
+
+// GET salary data by title, positionNumber, jobCode, or employeeId
+router.route('/salary/data')
```

Comment:
Not sure why this is '/salary/data' and not just '/salary'.  Also if no parameters are given, it returns a 404 - No records found.  Should it return all records?
```suggestion
router.route('/salary')
```